### ISSUE TITLE
loadFile doesn't accept Query Parameters with URL

### ISSUE THREAD

**Issue by pratik7span** [pichillilorenzo/flutter_inappwebview#1029]
- [:heavy_check_mark:] I have read the [Get Started - Installation](https://inappwebview.dev/docs/get-started/installation/) section
- [:heavy_check_mark:] I have read and done the [Get Started - Setup Android](https://inappwebview.dev/docs/get-started/setup-android/) section
- [:heavy_check_mark:] I have read and done the [Get Started - Setup iOS](https://inappwebview.dev/docs/get-started/setup-ios/) section
- [:heavy_check_mark:] I have already searched for the same problem

## Environment

| Technology           | Version       |
| -------------------- | ------------- |
| Flutter version      | 2.5.2 |
| Plugin version       | 5.3.2 |
| Android version      | not using |
| iOS version          | 15.0 |
| Xcode version        | 13 |

Device information: iPad mini 6th gen

## Description

**Expected behavior:** I want to pass query parameters with local file path. For example, `assets/board/index.html?letter=5&mode=1`

**Current behavior:** Like, currently I am passing `assets/board/index.html` and Its loading file successfully. but Its not working If I pass as above path (`assets/board/index.html?letter=5&mode=1`).

P.S. It is working for same URL if I host somewhere and use `initialUrlRequest`.

Do let me know if any other information you need. Kindly help.

**Comment by pratik7span** [comment]
> 👋 @pratik7span
> 
> **NOTE**: This comment is auto-generated.
> 
> Are you sure you have already searched for the same problem?
> 
> Some people open new issues but they didn't search for something similar or for the same issue. Please, search for it using the GitHub issue search box or on the official [inappwebview.dev](https://inappwebview.dev/) website, or, also, using Google, StackOverflow, etc. before posting a new one. You may already find an answer to your problem!
> 
> If this is really a new issue, then thank you for raising it. I will investigate it and get back to you as soon as possible. Please, make sure you have given me as much context as possible! Also, if you didn't already, post a code example that can replicate this issue.
> 
> In the meantime, you can already search for some possible solutions online! Because this plugin uses native WebView, you can search online for the same issue adding `android WebView [MY ERROR HERE]` or `ios WKWebView [MY ERROR HERE]` keywords.
> 
> Following these steps can save you, me, and other people a lot of time, thanks!

Thank you for instant auto-generated message. I tried everything everywhere on google. I found some solutions but that was different than what I want. I need it only when I load file from **assets with query paratmeters**.

**Comment by codercq** [comment]
I have the same problem, how can I solve it?  @pichillilorenzo

**Comment by Saikiran24** [comment]
Yes, I also have same problem loading files from Internal storage

**Comment by Jarvan1997** [comment]
Maybe you can use [In-App Localhost Server](https://inappwebview.dev/docs/5.x.x/in-app-localhost-server).

Like this:

`InAppWebView(
                initialUrlRequest: URLRequest(
                    url: Uri.parse("http://localhost:8080/assets/index.html?letter=5&mode=1")
                ),
                onWebViewCreated: (controller) {},
                onLoadStart: (controller, url) {},
                onLoadStop: (controller, url) {},
              ),`

**Comment by HadesPTIT** [comment]
I have the same problem, how can I solve it?

**Comment by ivkumar** [comment]
did anyone got solution for this?
